tests/e2e/vsan_test.go

```diff
@@ -36,6 +38,7 @@ import (
 )
 
 const (
+	vsanTest       = "vsan_test"
 	vsanVolumeTest = "vsan_vol"
 	vsanPolicyFlag = "vsan-policy-name"
```

Comment:
I think we only need to put const which need to me shared across the package to /tests/constants/...